composer/workflows/dataflowtemplateoperator_tutorial.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START composer_dataflow_dag]
+
+
+# Example Airflow DAG that creates a Cloud Dataproc cluster, runs the Hadoop
+# wordcount example, and deletes the cluster.
+
+# This DAG relies on three Airflow variables
+# https://airflow.apache.org/concepts.html#variables
+# * project_id - Google Cloud Project ID to use for the Cloud Dataflow cluster.
```

Comment:
These comments disagree with the variable names you are getting with modesl.Variable.get- can you update them to make them match?